aiinfra-cluster/outputs.tf

```diff
@@ -0,0 +1,20 @@
+/**
+  * Copyright 2022 Google LLC
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+output "dashboard_instructions" {
```

Comment:
where are you using this ? Do we need this output ?

Reply:
The end of the output of `create` now looks like this:
```
Apply complete! Resources: 2 added, 1 changed, 1 destroyed.

Outputs:

dashboard_instructions = <<EOT
A monitoring dashboard has been created. To view, navigate to the following URL:
https://console.cloud.google.com/monitoring/dashboards/builder/7668bc81-63d3-44da-92ce-9d5fc16567ac

EOT
Terraform apply finished successfully.
Jupyter notebook endpoint not available yet. Sleeping 15 seconds.
Jupyter notebook endpoint not available yet. Sleeping 15 seconds.
Cluster will be available after container exits.
=======================GPU cluster Information========================
Terraform state file location: gs://aiinfra-terraform-borisko-test/borisko-deployment/terraform/state
Parameter file location: gs://aiinfra-terraform-borisko-test/borisko-deployment/borisko-env.list
 Use below links to connect to the VMs: 
borisko-vm-7n7x : https://471b7a891726f1de-dot-us-central1.notebooks.googleusercontent.com
=======================================================================
Cluster provisioning successful.. 
Terraform cleanup is not needed.
```

Do you think that looks okay?

Reply:
can you please make sure it does not error out when enable-opsagent=false ?

Reply:
yup! when that is set, it looks like this:
```
Apply complete! Resources: 2 added, 0 changed, 0 destroyed.

Outputs:

dashboard_instructions = "Dashboard not created"
Terraform apply finished successfully.
Jupyter notebook endpoint not available yet. Sleeping 15 seconds.
Jupyter notebook endpoint not available yet. Sleeping 15 seconds.
Cluster will be available after container exits.
=======================GPU cluster Information========================
Terraform state file location: gs://aiinfra-terraform-borisko-test/borisko-deployment/terraform/state
Parameter file location: gs://aiinfra-terraform-borisko-test/borisko-deployment/borisko-env.list
 Use below links to connect to the VMs: 
borisko-vm-xqct : https://177d52a1efd45b58-dot-us-central1.notebooks.googleusercontent.com
=======================================================================
Cluster provisioning successful.. 
Terraform cleanup is not needed.
```

Reply:
awesome